components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -577,6 +577,7 @@ void RewardsServiceImpl::OnWalletInitialized(ledger::Result result) {
 
 void RewardsServiceImpl::OnWalletProperties(ledger::Result result,
     std::unique_ptr<ledger::WalletInfo> wallet_info) {
+  GetCurrentBalanceReport();
```

Comment:
I think this is an overkill. Can we just do it where is necessary. So when you claim a grant, do a tip and do a contribution.

Reply:
Updated to only occur in those scenarios, doing so on reconcile complete covers both tips and contributions.

Reply:
thank you